distribution/docs/src/main/resources/_contents/quickstart-contents.adoc

```diff
@@ -10,6 +10,7 @@ These steps will demonstrate:
 ==== Prerequisites
 
 * Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+** The Recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or above.
```

Comment:
I like "or later" (like you have in the other file) more than "or above". Also, "recommended" shouldn't be capitalized.